sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/rule/GatewayFlowRule.java

```diff
@@ -31,7 +31,8 @@
 
     private int grade = RuleConstant.FLOW_GRADE_QPS;
     private double count;
-    private long intervalSec = 1;
+    private long interval = 1;
```

Comment:
This could bring breaking changes. In fact, we could just convert the time unit in the dashboard side (convert to seconds), and keep `intervalSec` here.

Reply:
I also think about it, but if keep intervalSec here, though we can convert in dashboard side, we can't display the unit which are choosen by user.
e.g.
If user input 2, choose hour unit, convert to 7200 second in gateway rule. 
Then user refresh the list, and edit the rule, we can only get 7200,
choose 7200 second or 2 hour will get the same 7200 in this way.

I found that the `getIntervalSec` only used in serveral place, and change them to `calIntervalSec()`.

Reply:
It has been done. Keep intervalSec in `GatewayFlowRule` and convert in the dashboard side.
In `GatewayFlowRuleEntity` class, two methods `calIntervalSec`,`parseIntervalSec` are added to do convert.